src/constants/index.ts

```diff
@@ -1,4 +1,17 @@
 export const githubRepoURL = "https://github.com/tailcallhq/tailcall"
+export const tailCallBenchmarkUrl = "https://github.com/tailcallhq/graphql-benchmarks#benchmark-results"
+export const tailCallBlogUrl = "https://blog.tailcall.run/"
+export const codeSandboxUrl = "https://codesandbox.io/p/github/tailcallhq/tailcall-sandbox/main"
+export const zapierLink = "https://hooks.zapier.com/hooks/catch/2793322/3a1gxp2/"
+
+export const pageLinks = {
```

Comment:
move this to separate file routes.ts so that ppl know if they need to add a new route where to go

Reply:
added the page links in routes file